app/components/Feed/renders/adminRegistration.js

```diff
@@ -8,7 +8,10 @@ import type { AggregatedActivity } from '../types';
 /**
  * Normal grouping by target and date
  */
-export function activityHeader(aggregatedActivity: AggregatedActivity) {
+export function activityHeader(
+  aggregatedActivity: AggregatedActivity,
+  htmlTag: Function => string
```

Comment:
Isn't the flow typing for `htmlTag` here supposed to be something like:
```jsx
htmlTag: ({
  link: string,
  text: string,
  notLink: boolean
}) => React.Node<*>
```

Since you use this a lot, you can create a custom type ala:

```j
type MyType = {
  link: string,
  text: string,
  notLink: boolean
};